test/Dialect/Handshake/func.mlir

```diff
@@ -29,3 +29,15 @@ handshake.func public @public_func(%arg0 : i32, %ctrl: none) -> (i32, none) {
 handshake.func public @no_none_type() {
   return
 }
+
+// -----
+
+// CHECK-LABEL:   handshake.func @external(
+// CHECK-SAME:      i32, none, ...) -> none attributes {argNames = ["arg0", "ctrl"], resNames = ["outCtrl"]}
+handshake.func @external(%arg0: i32, %ctrl: none, ...) -> none
+
+// ----
+
+// CHECK-LABEL:   handshake.func @no_ssa_names(
+// CHECK-SAME:      i32, none, ...) -> none attributes {argNames = ["in0", "inCtrl"], resNames = ["outCtrl"]}
+handshake.func @no_ssa_names(i32, none, ...) -> none
```

Comment:
Considering that we've removed the requirement for handshake functions to have a mandatory none-typed control input, shouldn't the last argument here be inferred as in1 instead of inCtrl?

Reply:
That would make sense.
As this functionality reuses an existing function to generate the names, changing it might affect many tests. I'll check how many and submit another PR to change this.

Reply:
Awesome! ... and likewise for `outCtrl`.

Reply:
That's the same function ;P